src/components/scenes/Loans/ManageCollateralScene.tsx

```diff
@@ -302,7 +303,11 @@ export const ManageCollateralScene = <T extends keyof ParamList>(props: Props<T>
         />
         {showAprChange ? <AprCard apr={newDebtApr} key="apr" /> : null}
         <EdgeText style={styles.textTitle}>{opTypeStringMap[actionOpType].srcDestCard}</EdgeText>
-        <TappableAccountCard emptyLabel={s.strings.loan_select_receiving_wallet} selectedAsset={selectedAsset} onPress={handleShowWalletPickerModal} />
+        {bankAccountsMap != null ? (
```

Comment:
I meant actually for the loader to live in the WalletList itself in the Wallet Picker, but this is fine for now.